src/coreModules/notifications/components/Flash.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Message } from 'semantic-ui-react'
+
+import { withI18n } from 'coreModules/i18n/higherOrderComponents'
+
+const propTypes = {
+  descriptionKey: PropTypes.string,
```

Comment:
Add componentOption style (or other name of the key) which is an enum of ['info', 'success', 'warn', 'error']